otlp/traces_test.go

```diff
@@ -1258,67 +926,3 @@ func TestEvaluateSpanStatus(t *testing.T) {
 		})
 	}
 }
-
-func TestJsonRequest(t *testing.T) {
```

Comment:
This test seems thoroughly covered by the `application/json` encoding cases in `TestTranslateHttpTraceRequest` also updated in this file in this PR.